packages/sdk/src/browser-wallets/magiceden/index.ts

```diff
@@ -82,22 +87,18 @@ async function signPsbt(
 }
 
 async function signMessage(
+  getMeProvider: () => Promise<BitcoinProvider>,
   message: string,
   address: string,
   network: BrowserWalletNetwork = "mainnet",
 ): Promise<BrowserWalletSignResponse> {
-  if (!isInstalled()) {
+  if (!isInstalled(getMeProvider)) {
     throw new BrowserWalletNotInstalledError(
       "Magic Eden not installed or set as prioritised wallet.",
```

Comment:
```suggestion
      "Magic Eden not installed.",
```

https://github.com/ordzaar/ordit-sdk/pull/141/files#r1507341367